src/helpers/getPaginationLink.js

```diff
@@ -1,3 +1,11 @@
-export default (page, qs) => {
-  return `/petitions/page/${page}${qs ? `?${qs}` : ''}`;
+export default (params, page, qs) => {
+  var petitionsPath = '/petitions/';
+
+  if (params.cityName && params.city) {
+    petitionsPath += `${params.cityName}-${params.city}/`;
+  }
+
+  petitionsPath += `page/${page}${qs ? `?${qs}` : ''}`;
+
+  return petitionsPath;
 };
```

Comment:
I've just realised we have come this far without ever using the `path` helper from node, to generate paths 😬